lib/l10n/app_nb_NO.arb

```diff
@@ -1,6 +0,0 @@
-{
-  "cancel": "Avbryt",
-  "@cancel": {},
-  "reply": "Svar",
-  "@reply": {}
-}
```

Comment:
it is so weird for this PR

Reply:
These files are empty from the first place, and needed to be removed for the builds to succeed